### PR TITLE
chore(TripDetails): Tweak i18n message formatting.

### PR DESCRIPTION
**Pull request by binh-dam-ibigroup**
This PR cleans up the formatting of i18n messages following a request in https://github.com/opentripplanner/otp-react-redux/pull/508 (e.g. https://github.com/opentripplanner/otp-react-redux/pull/508#pullrequestreview-820355382).

This PR only contains ~a~multiple `chore`/`test` commits for the format changes and will not trigger a package release.

### CODE REVIEW

**Review by philip-cline — APPROVED** [review]

File: packages/trip-details/i18n/fr.yml [hunk 1]
```diff
@@ -2,11 +2,12 @@
 otpUi:
   TripDetails:
     calories: "Calories dépensées : <strong>{calories, number, ::.}</strong>"
-    caloriesDescription: "La dépense calorique est calculée sur la base de
+    caloriesDescription: >
+      La dépense calorique est calculée sur la base de
       {walkMinutes, plural, one {<strong># minute</strong> effectuée} other {<strong># minutes</strong> effectuées}} à pied et
       {bikeMinutes, plural, one {<strong># minute</strong> effectuée} other {<strong># minutes</strong> effectuées}} en vélo sur ce trajet.
       (d'après <dietaryLink>Dietary Guidelines for Americans 2005, page 16, Table 4</dietaryLink>)"
```
Comment (fpurcell) [inline packages/trip-details/i18n/fr.yml:9]: Typo:  the trailing quote mark at end of this string.

Comment (fpurcell) [reply packages/trip-details/i18n/fr.yml:9]: Screen Shot 2021-12-03 at 1 12 50 PM

**Comment by fpurcell** [comment]
Looks good, Binh ... just remove the typo.

**Review by fpurcell — APPROVED** [review]

File: packages/trip-details/i18n/fr.yml (hunk 1 above)
Comment (binh-dam-ibigroup) [reply packages/trip-details/i18n/fr.yml]: Thanks @fpurcell for spotting that one! Handled in https://github.com/opentripplanner/otp-ui/pull/321/commits/6850675cbd2c7a688d6e2a5ef50bac418f0fa03d.

**Review by miles-grant-ibigroup — APPROVED** [review]
This looks great! Thanks so much for doing this.

**Pull request merged**